scripts/contractInteraction/amm.py

```diff
@@ -345,6 +346,28 @@ def withdrawFromRBTCWrapperProxy(tokenAddress, to, amount):
     wrapperProxy = Contract.from_abi("RBTCWrapperProxy", address=conf.contracts['RBTCWrapperProxy'], abi=abi, owner=conf.acct)
     wrapperProxy.withdraw(tokenAddress, to, amount)
 
+def transferOwnershipAMMContractsToGovernance(contractAddress, destinationAddress, contractName=''):
+    abiFile =  open('./scripts/contractInteraction/ABIs/Owned.json')
+    abi = json.load(abiFile)
+    ammContract = Contract.from_abi("AMMContract", address=contractAddress, abi=abi, owner=conf.acct)
+
+    if(contractName):
+        # # verify the contract address
+        contractRegistry = Contract.from_abi("sovryn", address=conf.contracts['ammContractRegistry'], abi=interface.IContractRegistry.abi, owner=conf.acct)
+        _contractAddress = contractRegistry.addressOf(web3.toHex(contractName.encode('utf-8')).ljust(66, '0'))
+
+        if(_contractAddress != contractAddress):
+            raise Exception("Unmatched contract address with the on-chain")
+
+    currentOwner = ammContract.owner()
+    if(currentOwner != conf.contracts['multisig']):
+        raise Exception("Multisig is not the owner")
+
+    print("Transferring ownership of {0} to {1}".format(contractAddress, destinationAddress))
+    data = ammContract.transferOwnership.encode_input(destinationAddress)
```

Comment:
```suggestion
    print("Transferring ownership of {0} to {1}".format(contractAddress, newOwnerAddress))
    data = ammContract.transferOwnership.encode_input(newOwnerAddress)
```

Reply:
Updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/450/commits/94459ccdbffb4c8e2fa42981a355178c076b9a2f)